checker/check_result_test.go

```diff
@@ -127,6 +127,14 @@ func TestCreateProportionalScore(t *testing.T) {
 			},
 			want: 5,
 		},
+		{
+			name: "2 and 5",
+			args: args{
+				success: 2,
+				total:   5,
+			},
+			want: 4,
```

Comment:
i'm curios, did the cover something that wasn't before?

Reply:
I added it for local testing during this migration. I don't think it added more coverage, but it adds a score that did not exist before.